### ISSUE TITLE
Backspace is not clears the date in calendar.

### ISSUE THREAD

**Issue by yigitfindikli** [primefaces/primeng#7897]
```
[ x] bug report => Search github for a similar issue or PR before submitting
[ ] feature request => Please check if request is not on the roadmap already https://github.com/primefaces/primeng/wiki/Roadmap
[ ] support request => Please do not submit support request here, instead see http://forum.primefaces.org/viewforum.php?f=35
```

**Current behavior**
Backspace is not clears the date.

**Expected behavior**
Backspace should clear the date.

**Minimal reproduction of the problem with instructions**
It can be replicate in showcase. https://www.primefaces.org/primeng/#/calendar

* **Angular version:** 7+

* **PrimeNG version:** 7+

**Comment by JacobSiegle** [comment]
Duplicate I think - 
https://github.com/primefaces/primeng/issues/7810

Possibly fixes - 
https://github.com/primefaces/primeng/issues/7683

**Comment by SaskiaLue** [comment]
Any news on this? We're using this component quite extensively and do need the ability to clear dates.
Thanks

**Comment by JacobSiegle** [comment]
It says in the commits above it's fixed in 8.0.1.
